### DISCUSSION TITLE
Change position of label in chart treemap

### DISCUSSION THREAD

**Discussion by ghost** [apexcharts/apexcharts.js#4163] [General]
How do i change position of label in chart treemap apexcharts?
![image](https://user-images.githubusercontent.com/59729672/193999089-cfff94eb-3ea4-415b-98b4-61daa2cdca81.png)

**Comment by cue28** [comment]
Same question from 2023! Did you solve it?

**Comment by ghost** [comment]
> Same question from 2023! Did you solve it?

Yes

**Reply by cue28** [reply]
How did you do that?

**Reply by raAndrievskiy** [reply]
I am facing the same task, but I do not understand how to realize it, I will be grateful for help

**Reply by RodrigoRVSN** [reply]
Anyone found a solution for this?
